lib/Compile.js

```diff
@@ -168,56 +188,172 @@ var Compile = module.exports = function (options) {
             var styles;
             var opacity;
             var path;
+            
+            //var printerCutLineGroup = document.createElement("g");
+            
+            var printerCutline = document.createElement("polygon");
+            var cutlinePoints = "";
+            
+            if(options.stickerSheet) {
+              printerCutline.setAttribute("style", "fill:none;stroke:#FF00FF;stroke-width:1");
+            }
             Geometry.all.forEach(function(obj, key) {
-                  
-                  styles = '';
-                  path  = document.createElement("path");
-                  
-                  // Theres an issue with Y being flipped, maybe to do with viewBox so * -1 seems to work fancy like!
-                  // Create one trixel
-                  path.setAttribute('d', 'M ' + obj.v0.x * unitScale + ' '+ (obj.v0.y * unitScale)*-1 +'  L ' + obj.v1.x * unitScale + ' '+ (obj.v1.y * unitScale)*-1 +' L ' + obj.v2.x * unitScale + ' '+ (obj.v2.y * unitScale)*-1 +'z');
-                  
-                  // Pulling the color from the JSON data
-                  if (jsonData[key]) {
-                    
-                      if (typeof jsonData[key].color !== 'undefined') {
-                        styles += 'fill:'+jsonData[key].color+';';
-                      }
-                      
-                      if (typeof jsonData[key].opacity !== 'undefined' && jsonData[key].opacity != 1) {
-                        styles += 'fill-opacity:'+jsonData[key].color+';';
-                      }
-                      
-                      path.setAttribute('style',  styles);
-                  } else {
-                    
-                      path.setAttribute('style',  'fill:'+randomRGB()+';');
+              
+              if(options.stickerSheet) {
```

Comment:
Yeah I see that there's a very clear correlation between total trixels / number of layers and how many trixels per layer. It would just take some time to generalize it but yes it needs to be done.